src/dmd/link.d

```diff
@@ -1053,12 +1048,15 @@ version (Windows)
                     const pathlen = strlen(path);
                     const addpathlen = strlen(addpath);
 
-                    char* npath = cast(char*)mem.xmalloc(5 + pathlen + 1 + addpathlen + 1);
+                    const length = 5 + addpathlen + 1 + pathlen;
+                    char* npath = cast(char*)mem.xmalloc(length + 1);
                     memcpy(npath, "PATH=".ptr, 5);
                     memcpy(npath + 5, addpath, addpathlen);
                     npath[5 + addpathlen] = ';';
                     memcpy(npath + 5 + addpathlen + 1, path, pathlen + 1);
-                    putenv(npath);
+                    if (0 != LocalEnvVar(cast(string)npath[0 .. length], 4).putenvRestorable)
```

Comment:
please don't use the yodel style

Reply:
Actually in this cast, `putenvRestorable` is a member of `LocalEnvVar`

Reply:
I meant remove the `0 !=`

Reply:
gotcha, done